keyboards/varanidae/info.json

```diff
@@ -0,0 +1,151 @@
+{
+    "keyboard_name": "Varanidae",
+    "manufacturer": "Tokage IT Lab.",
+    "url": "https://github.com/TokageItLab/Varanidae",
+    "maintainer": "Silc Renew",
+    "usb": {
+        "vid": "0x5369",
+        "pid": "0x0001",
+        "device_version": "1.0.0"
+    },
+    "pin_compatible": "elite_c",
```

Comment:
This is not really correct, because the definition of `elite_c` compatibility in QMK does not include the `B0` pin of Elite-C, and your PCB uses that pin.  However, there is no way to explicitly set “none” here (only the `"promicro"` and `"elite_c"` values are allowed by the JSON schema), and `"development_board": "elite_c"` implicitly sets `"pin_compatible": "promicro"` for historical reasons.  So maybe `"pin_compatible": "elite_c"` would need to stay here for now (but the code won't compile anyway if a converter is selected, because the converters don't provide a mapping for the `B0` pin).

Reply:
Using the B0 pin is currently unavoidable and must be left as is.